airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresSourceTest.java

```diff
@@ -481,8 +481,7 @@ void testGetDefaultConnectionPropertiesWithSsl() {
     final Map<String, String> defaultConnectionProperties = new PostgresSource().getDefaultConnectionProperties(config);
     assertEquals(defaultConnectionProperties, ImmutableMap.of(
         "ssl", "true",
-        "sslmode", "require"
-    ));
+        "sslmode", "require"));
```

Comment:
this change looks unnecessary

Reply:
these changes were added automatically after running the code formatting command. I left them so that later developers who will do automatic formatting of the code do not encounter these changes.